setup.py

```diff
@@ -21,8 +21,10 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 """
+from __future__ import print_function
```

Comment:
hmm, ok I never bothered importing this before. maybe six does it implicitly, or python 2.7 just was able to deal with the python3 syntax as I had it

Reply:
same as above comment, I don't think it is needed